familias/serializers.py

```diff
@@ -0,0 +1,280 @@
+from rest_framework import serializers
+
+from estudios_socioeconomicos.utils import save_foreign_relationship
+
+from .models import Familia, Comentario, Integrante, Alumno, Tutor
+
+
+class ComentarioSerializer(serializers.ModelSerializer):
+    """ Serializer to represent a .models.Comentario instance
+        through a REST endpoint for the offline application
+        to submit information.
+    """
+    class Meta:
+        model = Comentario
+        fields = ('id', 'fecha', 'texto')
+        extra_kwargs = {"id": {"read_only": False, "required": False}}
+
+    def create(self, family):
+        """ This function overides the default behaviour for creating
+            an object through a serializer.
+
+            This serializer dependes on a .models.Familia
+            object instance. Since we are dealing with
+            nested created objects, the familia must be
+            created first and passed as parameter to the
+            created function.
+        """
+        self.validated_data['familia'] = family
+        return Comentario.objects.create(**self.validated_data)
+
+    def update(self):
+        """ This function overides the default behaviour for creating
+            an object through a serializer since Nested Updates are
+            not implemented by DRF.
+
+            Returns
+            -------
+            Updated Instance of Comentario model.
+        """
+        Comentario.objects.filter(pk=self.instance.id).update(**self.validated_data)
```

Comment:
You are right, it wasn't that needed because it does not have nested object, but to be able to call all the updates from same function, the definition of update changed so this update method just for it to fit the others.